stellargraph/core/graph.py

```diff
@@ -182,17 +203,17 @@ class StellarGraph:
     .. seealso:: :meth:`from_networkx` for construction from a NetworkX graph.
 
     Args:
-        nodes (DataFrame or dict of hashable to Pandas DataFrame, optional):
-            Features for every node in the graph. Any columns in the DataFrame are taken as numeric
-            node features of type ``dtype``. If there is only one type of node, a DataFrame can be
+        nodes (Numpy array, RowFrame, DataFrame or dict of hashable to RowFrame or Pandas DataFrame, optional):
+            Features for every node in the graph. The values are taken as numeric
+            node features of type ``dtype``. If there is only one type of node, a NumPy array, :class:`RowFrame` or DataFrame can be
             passed directly, and the type defaults to the ``node_type_default`` parameter. Nodes
             have an ID taken from the index of the dataframe, and they have to be unique across all
-            types.  For nodes with no features, an appropriate DataFrame can be created with
-            ``pandas.DataFrame([], index=node_ids)``, where ``node_ids`` is a list of the node
+            types.  For nodes with no features, an appropriate value can be created with
+            ``RowFrame(index=node_ids)``, where ``node_ids`` is a list of the node
             IDs. If this is not passed, the nodes will be inferred from ``edges`` with no features
             for each node.
 
-        edges (DataFrame or dict of hashable to Pandas DataFrame, optional):
+        edges (Numpy array, RowFrame, DataFrame or dict of hashable to RowFrame or Pandas DataFrame, optional):
```

Comment:
from what I can tell, I don't think `edges` can be constructed from a `RowFrame` because it needs column names

Reply:
Oops, copy-paste mistake.